model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -705,13 +705,10 @@ def _create_default_perf_analyzer_config(
             model_config.get_field("name"), self._config
         )
 
-        default_concurrency = self._calculate_default_concurrency(model_config)
-
-        perf_config_params = {
-            "batch-size": DEFAULT_BATCH_SIZES,
-            "concurrency-range": default_concurrency,
-        }
-        default_perf_analyzer_config.update_config(perf_config_params)
+        if not "request-intervals" in model.perf_analyzer_flags():
```

Comment:
Doing a self-review: I'm wondering if this should be `if not model.is_load_specified()` just like line 515